src/ProjectSystemTools/TableControl/ElapsedColumnDefinition.cs

```diff
@@ -25,11 +25,11 @@ internal sealed class ElapsedColumnDefinition : TableColumnDefinitionBase
 
         public override bool TryCreateStringContent(ITableEntryHandle entry, bool truncatedText, bool singleColumnView, out string content)
         {
-            if (entry.TryGetValue(TableKeyNames.Status, out var statusValue) && statusValue != null && statusValue is BuildStatus status &&
-                status != BuildStatus.Running &&
-                entry.TryGetValue(TableKeyNames.Elapsed, out var value) && value != null && value is TimeSpan timeSpan)
+            if (entry.TryGetValue(TableKeyNames.Status, out var statusValue) && statusValue != null && statusValue is BuildStatus status
+                && status != BuildStatus.Running
+                && entry.TryGetValue(TableKeyNames.Elapsed, out var value) && value != null && value is TimeSpan timeSpan)
             {
-                content = timeSpan.ToString("g");
+                content = timeSpan.ToString(@"hh\:mm\:ss\:ff");
```

Comment:
This change separates seconds from the fractional part with a colon rather than a period, which is likely confusing.

I read #31 as suggesting removing the `hh:mm` components. Consider using `timeSpan.TotalSeconds.ToString("N3")` instead, so the fractional part covers milliseconds and so the separator is locale specific.